src/components/atoms/ErrorMessage.tsx

```diff
@@ -0,0 +1,29 @@
+import { InlineAlert } from 'evergreen-ui';
+import React from 'react';
+import { Text, Pane } from 'evergreen-ui';
+
+export type ErrorMessageProps = {
+    size?: 300 | 400 | 500 | 600;
+    message?: string;
+    moreInfo?: string;
+};
+
+const ErrorMessage: React.FC<ErrorMessageProps> = ({ message = 'Error :(', moreInfo, size = 400 }) => {
```

Comment:
A nifty trick is to replace the 

```
{
  return (
```

with just `(` (meaning implicit return).

You don't need to change this, but it is a helpful tool.